.github/workflows/macos_wheel.yaml

```diff
@@ -32,7 +62,12 @@ jobs:
         python-version: ['3.9']
         ffmpeg-version-for-tests: ['4.4.2', '5.1.2', '6.1.1', '7.0.1']
     if: ${{ always() }}
+    needs: build
     steps:
+      - uses: actions/download-artifact@v3
+        with:
+          name: pytorch_torchcodec__${{ matrix.python-version }}_cpu_
+          path: pytorch/torchcodec/dist/
       - name: Setup conda env
```

Comment:
Looks like this step is not working as intended. I get something like this:
```
....
  # To activate this environment, use
  #
  #     $ conda activate test
  #
  # To deactivate an active environment, use
  #
  #     $ conda deactivate
  

setup-miniconda ran successfully
```

However next step:
```
Run python3 -m pip install --upgrade pip
  python3 -m pip install --upgrade pip
  shell: /bin/bash -l -eo pipefail {0}
  env:
    INPUT_RUN_POST: true
    CONDA: /Users/ec[2](https://github.com/pytorch/torchcodec/actions/runs/11260601615/job/31315172014?pr=230#step:4:2)-user/miniconda3
    CONDA_PKGS_DIR: /Users/ec2-user/conda_pkgs_dir
Requirement already satisfied: pip in /Library/Frameworks/Python.framework/Versions/[3](https://github.com/pytorch/torchcodec/actions/runs/11260601615/job/31315172014?pr=230#step:4:3).10/lib/python3.10/site-packages (24.2)
```

It picks python 3.10